.pylintrc

```diff
@@ -5,7 +5,7 @@
 # A comma-separated list of package or module names from where C extensions may
 # be loaded. Extensions are loading into the active Python interpreter and may
 # run arbitrary code.
-extension-pkg-whitelist=
+extension-pkg-whitelist=ujson
```

Comment:
only needed for tests.  not actually needed since it's a benchmark test, can remove if desired